### PR TITLE
Updated FAQ to reflect current curriculum

### PR DESCRIPTION
**Pull request by programmurr**
Updated the FAQ https://github.com/TheOdinProject/theodinproject/issues/1755

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -59,15 +59,16 @@ def faq_items
...
           <br />
-          <p>The Odin Project takes a back-to-front approach to learning Ruby on Rails.  We start with a <a href='courses/ruby-programming'>strong foundation in Ruby</a> and an understanding of databases before migrating into <a href='courses/ruby-on-rails'>Ruby on Rails</a> (which is a framework built using Ruby).</p>
+          <p> The Full Stack Javascript path maintains our front-end momentum from Foundations by delving into <a href='courses/javascript'>Javascript.</a> Once you've got a solid foundation here, you'll take a deeper look at front-end design with <a href='courses/html-and-css'>HTML and CSS</a> before taking your JavaScript skills to the server-side with Express and MongoDB in <a href='courses/nodejs'>NodeJS</a>. </p>
```
Comment (KevinMulhern) [inline app/helpers/application_helper.rb:66]: 💯  mentioning mongo is a great addition.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -59,15 +59,16 @@ def faq_items
...
+          <p> Along the way in either path you’ll learn Git and deployment so you can build your own portfolio, collaborate with others, and see your work on the web.Finally, The Odin Project will provide you with opportunities to work with legacy codebases and open-source projects. We’ll also cover areas specifically intended to help when you’re <a href='courses/getting-hired'>looking for employment</a>.</p>
           <br />
-          <p>Finally, The Odin Project will provide you with opportunities to work with legacy codebases and open-source projects. We’ll also cover areas specifically intended to help when you’re <a href='courses/getting-hired'>looking for employment</a>.</p>"
+          <p> As you may have noticed, there is considerable overlap between the two paths with the only difference between them being the choice of back-end technology. They both guide you through harnessing the power of <a href='https://reactjs.org/'>React</a> along with software development paradigms such as Object-Oriented Programming and Test-Driven Development. More importantly, both paths will equally allow you to develop your own skills as a fullstack web developer. </p>
```
Comment (KevinMulhern) [inline app/helpers/application_helper.rb:70]: Minor typo at "see your work on the web.Finally," Just needs a space after the full stop.

**Review by KevinMulhern — COMMENTED** [review]
Very nice work @programmurr 💪  this reads much better than what we had before and perfectly describes the current path set up.

Just one very minor typo.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (programmurr) [reply app/helpers/application_helper.rb:70]: Just pushed the amended line

**Review by KevinMulhern — APPROVED** [review]
Looks great to me, thanks for doing this @programmurr 🥳

**Pull request merged**